src/index.html

```diff
@@ -11,6 +11,43 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="post"
+      class="search__form search__form-big"
+      data-qa="big"
+      required
+    >
+      <button class="search__button-big">
+        <img src="./images/Search.svg" alt="Search">
+      </button>
```

Comment:
No need to add this img to html, set it as background through css
Here's an example:
https://codepen.io/adskiy/pen/mdmLqXB